### ISSUE TITLE
pygraphiz appears to be deprecated?

### ISSUE THREAD

**Issue by psychemedia** [Alexis-benoist/eralchemy#46]
`pygraphviz` looks to be no longer maintained. Would it be worth migrating `eralchemy` to use [`graphviz`](https://github.com/xflr6/graphviz)?

```
#Rendering from dot file using graphviz
from graphviz import Source
Source.from_file(dotfilename)
```

The `format` parameter to `Source` defines the output format and the `engine` parameter the layout engine (*dot*, *neato* etc).

The only place `pygraphviz` appears to be used is in eralchemy/eralchemy/main.py in the function:

```
def intermediary_to_schema(tables, relationships, output):
    """ Transforms and save the intermediary representation to the file chosen. """
    dot_file = _intermediary_to_dot(tables, relationships)
    graph = AGraph()
    graph = graph.from_string(dot_file)
    extension = output.split('.')[-1]
    graph.draw(path=output, prog='dot', format=extension)
```

**Comment by Alexis-benoist** [comment]
Hey, 

I have nothing against that, as long as it's tested :)

**Comment by remram44** [comment]
Outputting the DOT source would be sufficient for me. Having to get the graphviz development headers seem unnecessary when I can just run the `dot` program.

**Comment by choj** [comment]
I might have done this per https://github.com/Alexis-benoist/eralchemy/issues/47#issuecomment-584834753

**Comment by Alexis-benoist** [comment]
If someone is willing to migrate it, I'm not against :)

**Comment by jaroli** [comment]
Same problem here, cannot install eralchemy due to the pygraphviz package

```
apt install graphviz
pip install eralchemy
...
Building wheels for collected packages: pygraphviz
  Building wheel for pygraphviz (setup.py): started
  Building wheel for pygraphviz (setup.py): finished with status 'error'
  ERROR: Command errored out with exit status 1:
```

**Comment by 12rambau** [comment]
This error seems pretty old and I just checked https://github.com/pygraphviz/pygraphviz/graphs/code-frequency and it seems perfectly maintained (last commit was 2 weeks ago). I assume the installation problems mentioned in 2020 are now solved. I'll close it for now as we are refactoring the package to include latest @maurerle modification from `eralchemy2` don't hesitate to request reopening if you really think pygraphviz should be dropped.
